src/app/component/general/eco-news/create-news/create-news.component.html

```diff
@@ -9,8 +9,8 @@ <h2>Create news</h2>
     </div>
     <div class="info">
```

Comment:
Please revert this file, it's out of the scope of your ticket

Reply:
fixed